src/App.svelte

```diff
@@ -586,11 +604,56 @@
 			</div>
 		</div>
 	</div>
-	<div class="section" id="sponsors">
+	
+	<div style="background-color: var(--background); height:10rem; padding-left: 7.5rem;" >
 		<h1>Sponsors</h1>
-		<br>
-		<h2>More coming soon!</h2>
-		<br>
+		<h3>Interested in sponsoring HackKU23?</h3>
+		<h3>Email us at <a href="mailto:sponsorship.hackku@gmail.com">sponsorship.hackku@gmail.com</a></h3>
+	</div>
+	<div class="section" id="sponsors" style="row-gap:4rem; align-items:center;">
+		
+		<div class="sponsor-tiers">
+			<h2>PETA Tier</h2>
+			<div class="sponsor-image-holders">
+				<img src="security_benefit.png" alt="Security Benefit Logo" class="sponsor-image">
+			</div>
+		</div>
+		<div class="sponsor-tiers">
+			<h2>TERA Tier</h2>
+			<div class="sponsor-image-holders">
+				<img src="Ripple.png" alt="Ripple Logo" class="sponsor-image">
+			</div>
+		</div>
+		<div class="sponsor-tiers">
+			<h2>GIGA Tier</h2>
+			<div class="sponsor-image-holders">
+				<img src="Air Power Consultants.png" alt="HRBlock Logo" class="sponsor-image" style="max-width:25rem">
+				<img src="HRBlock_Logo_250px.png" alt="HRBlock Logo" class="sponsor-image" style="max-width:25rem">
+				<img src="VeteransUnited-HORIZONTAL-[WHITE]-01 (5).png" alt="HRBlock Logo" class="sponsor-image" style="max-width:25rem">
+			</div>
+		</div>
+		<div class="sponsor-tiers">
+			<h2>MEGA Tier</h2>
+			<div class="sponsor-image-holders">
+				<img src="Tradebot Logo-flat-Registered Trademark.png" alt="TradeBot Logo" class="sponsor-image">
+			</div>
+		</div>
+		<div class="sponsor-tiers">
+			<h2>FOOD</h2>
+			<div class="sponsor-image-holders">
+				<img src="laestrella (1).png" alt="Laestralla Logo" class="sponsor-image" style="max-width:25rem">
+				<img src="mcclains.png" alt="Mcclains Logo" class="sponsor-image" style="max-width:25rem">
+				<img src="globeindianfood.png" alt="Globe Indian Logo" class="sponsor-image" style="max-width:25rem">
+			</div>
+		</div>
+		<div class="sponsor-tiers">
+			<h2>IN KIND</h2>
+			<div class="sponsor-image-holders">
+				<img src="GoogleCloud_FullColor 2900x512px.png" alt="Google Cloud Logo" class="sponsor-image" style="max-width:25rem">
+				<img src="echo3D - Logo 2021 - Dark.png" alt="Echo 3D Logo" class="sponsor-image" style="max-width:25rem">
+				<img src="balsamiq-logo-screen.png" alt="Balsamiq Logo" class="sponsor-image" style="max-width:25rem">
+			</div>
+		</div>
```

Comment:
could you add a screenshot (or screen recording if you have to scroll) of what this looks like?